modules/features2d/include/opencv2/features2d.hpp

```diff
@@ -529,6 +529,9 @@ class CV_EXPORTS_W GFTTDetector : public Feature2D
     CV_WRAP virtual void setBlockSize(int blockSize) = 0;
     CV_WRAP virtual int getBlockSize() const = 0;
 
+    CV_WRAP virtual void setGradiantSize(int gradiantSize) = 0;
```

Comment:
New virtual methods also break binary compatibility. You can declare them as non-virtual and implicitly cast `this` pointer to `GFTTDetector_Impl` to call the real implementation.